cmd/chaos-stress/main.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"math/rand"
+	"os"
+	"time"
+
+	"github.com/pingcap/chaos-mesh/pkg/chaosstress"
+	"github.com/pingcap/chaos-mesh/pkg/version"
+
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+var (
+	addr         string
+	printVersion bool
+	log          = ctrl.Log.WithName("chaos-stress")
+)
+
+func initFlag() {
+	flag.StringVar(&addr, "addr", ":65533", "RPC server address")
```

Comment:
Do we need to add chaos-stress to chaos-daemon.  @cwen0 @YangKeao @Yisaer what about your opinion

Reply:
No. `chaos-stress` is a sidecar container, but not a daemon.